dev/aarch64_opt/src/arith_native_aarch64.h

```diff
@@ -8,44 +8,46 @@
 #include <stdint.h>
 #include "../../../common.h"
 
-#define aarch64_ntt_zetas_layer12345 MLK_NAMESPACE(aarch64_ntt_zetas_layer12345)
-#define aarch64_ntt_zetas_layer67 MLK_NAMESPACE(aarch64_ntt_zetas_layer67)
+#define mlk_aarch64_ntt_zetas_layer12345 \
+  MLK_NAMESPACE(aarch64_ntt_zetas_layer12345)
+#define mlk_aarch64_ntt_zetas_layer67 MLK_NAMESPACE(aarch64_ntt_zetas_layer67)
 #define aarch64_invntt_zetas_layer12345 \
   MLK_NAMESPACE(aarch64_invntt_zetas_layer12345)
-#define aarch64_invntt_zetas_layer67 MLK_NAMESPACE(aarch64_invntt_zetas_layer67)
+#define mlk_aarch64_invntt_zetas_layer67 \
+  MLK_NAMESPACE(aarch64_invntt_zetas_layer67)
 #define aarch64_zetas_mulcache_native \
   MLK_NAMESPACE(aarch64_zetas_mulcache_native)
 #define aarch64_zetas_mulcache_twisted_native \
   MLK_NAMESPACE(aarch64_zetas_mulcache_twisted_native)
```

Comment:
why are those not namespaced?

Reply:
Because my script didn't detect definitions past line breaks :-(